src/main/java/http/HttpHeaderInfo.java

```diff
@@ -0,0 +1,50 @@
+package http;
+
+import com.github.jknack.handlebars.internal.lang3.StringUtils;
+import com.google.common.collect.Maps;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created By kjs4395 on 2020-06-05
+ */
+public class HttpHeaderInfo {
+    private Map<String, String> headerInfoMap;
+
+    public HttpHeaderInfo() {
+        this.headerInfoMap = new HashMap<>();
+    }
+
+    public void addHeaderValue(String headerLine) {
+        String[] values = headerLine.split(":",2);
+
+        if(values.length < 2) {
+            throw new IllegalArgumentException();
+        }
+
+        this.headerInfoMap.put(values[0].trim(), values[1].trim());
+    }
+
+    public void addKeyAndValue(String key, String value) {
+        if(this.headerInfoMap == null) this.headerInfoMap = new HashMap<>();
```

Comment:
공백에 대한 컨벤션 부분들은 IDE 의 reformat 기능 등을 사용하여 습관적으로 반영하시는 것이 좋다고 생각합니다.
```suggestion
        if (this.headerInfoMap == null) {
            this.headerInfoMap = new HashMap<>();
        }
```